<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="sukeerthsahana" failures="0" tests="16" name="dsAlgoTests.ArrayTest" time="21.042" errors="1" timestamp="2025-02-05T12:28:42 CST" skipped="15">
  <testcase classname="dsAlgoTests.ArrayTest" name="checkErrorMessageWhenExecutingWithoutCodeArrayUsingListForTryEditor" time="19.513">
    <skipped/>
  </testcase> <!-- checkErrorMessageWhenExecutingWithoutCodeArrayUsingListForTryEditor -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="PracticeQuestionVaildAndInvalidDataWithRunButton" time="0.000">
    <skipped/>
  </testcase> <!-- PracticeQuestionVaildAndInvalidDataWithRunButton -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="navigateToArrayUsingList" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToArrayUsingList -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="navigateToApplicationOfArray" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToApplicationOfArray -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="PracticeQuestionVaildAndInvalidDataWithSubmitButton" time="0.000">
    <skipped/>
  </testcase> <!-- PracticeQuestionVaildAndInvalidDataWithSubmitButton -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="navigateToArrayInPythonTryEditor" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToArrayInPythonTryEditor -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="navigateToApplicationOfArrayTryEditor" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToApplicationOfArrayTryEditor -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="verifyCodeExecutionForArraySections" time="0.000">
    <skipped/>
  </testcase> <!-- verifyCodeExecutionForArraySections -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="checkErrorMessageWhenExecutingWithoutCodeInArrayInPythonForTryEditor" time="0.000">
    <skipped/>
  </testcase> <!-- checkErrorMessageWhenExecutingWithoutCodeInArrayInPythonForTryEditor -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="checkErrorMessageWhenExecutingWithoutCodeApplicationofArrayForTryEditor" time="0.000">
    <skipped/>
  </testcase> <!-- checkErrorMessageWhenExecutingWithoutCodeApplicationofArrayForTryEditor -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="navigateToArrayUsingListTryEditor" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToArrayUsingListTryEditor -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="navigateToBasicOperationInList" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToBasicOperationInList -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="checkErrorMessageWhenExecutingWithoutCodeBasicOperationInListsForTryEditor" time="0.000">
    <skipped/>
  </testcase> <!-- checkErrorMessageWhenExecutingWithoutCodeBasicOperationInListsForTryEditor -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="navigateToArrayPage" time="1.529">
    <error message="Error communicating with the remote browser. It may have died.
Build info: version: &#039;4.17.0&#039;, revision: &#039;e52b1be057*&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;21.0.3&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [cfef21006b825265d09da7ca5a2778c7, findElement [using, value]]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 133.0.6943.53, chrome: {chromedriverVersion: 133.0.6943.53 (9a80935019b0..., userDataDir: C:\Users\sahan\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57004}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57004/devtoo..., se:cdpVersion: 133.0.6943.53, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: cfef21006b825265d09da7ca5a2778c7" type="org.openqa.selenium.remote.UnreachableBrowserException">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '4.17.0', revision: 'e52b1be057*'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.3'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [cfef21006b825265d09da7ca5a2778c7, findElement [using, value]]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 133.0.6943.53, chrome: {chromedriverVersion: 133.0.6943.53 (9a80935019b0..., userDataDir: C:\Users\sahan\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57004}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57004/devtoo..., se:cdpVersion: 133.0.6943.53, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: cfef21006b825265d09da7ca5a2778c7
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:546)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:356)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:350)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy14.click(Unknown Source)
at dsAlgoPages.HomePage.clickSignin(HomePage.java:71)
at dsAlgoTests.ArrayTest.navigateToArrayPage(ArrayTest.java:42)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.util.concurrent.RejectedExecutionException: Task jdk.internal.net.http.common.SequentialScheduler$SchedulableTask@65f0f9c4 rejected from java.util.concurrent.ThreadPoolExecutor@5dd599b2[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 45]
at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2081)
at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:841)
at java.base/java.util.concurrent.ThreadPoolExecutor.execute(ThreadPoolExecutor.java:1376)
at java.net.http/jdk.internal.net.http.HttpClientImpl$DelegatingExecutor.ensureExecutedAsync(HttpClientImpl.java:183)
at java.net.http/jdk.internal.net.http.HttpClientImpl$DelegatingExecutor.execute(HttpClientImpl.java:175)
at java.net.http/jdk.internal.net.http.common.SequentialScheduler.runOrSchedule(SequentialScheduler.java:282)
at java.net.http/jdk.internal.net.http.common.SequentialScheduler.runOrSchedule(SequentialScheduler.java:251)
at java.net.http/jdk.internal.net.http.Http1AsyncReceiver.onReadError(Http1AsyncReceiver.java:516)
at java.net.http/jdk.internal.net.http.Http1AsyncReceiver$Http1TubeSubscriber.onComplete(Http1AsyncReceiver.java:601)
at java.net.http/jdk.internal.net.http.SocketTube$InternalReadPublisher$ReadSubscription.signalCompletion(SocketTube.java:648)
at java.net.http/jdk.internal.net.http.SocketTube$InternalReadPublisher$InternalReadSubscription.read(SocketTube.java:853)
at java.net.http/jdk.internal.net.http.SocketTube$SocketFlowTask.run(SocketTube.java:181)
at java.net.http/jdk.internal.net.http.common.SequentialScheduler$SchedulableTask.run(SequentialScheduler.java:207)
at java.net.http/jdk.internal.net.http.common.SequentialScheduler.runOrSchedule(SequentialScheduler.java:280)
at java.net.http/jdk.internal.net.http.common.SequentialScheduler.runOrSchedule(SequentialScheduler.java:233)
at java.net.http/jdk.internal.net.http.SocketTube$InternalReadPublisher$InternalReadSubscription.signalReadable(SocketTube.java:782)
at java.net.http/jdk.internal.net.http.SocketTube$InternalReadPublisher$ReadEvent.signalEvent(SocketTube.java:965)
at java.net.http/jdk.internal.net.http.SocketTube$SocketFlowEvent.handle(SocketTube.java:253)
at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.handleEvent(HttpClientImpl.java:1467)
at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.lambda$run$3(HttpClientImpl.java:1412)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:1412)
]]>
    </error>
  </testcase> <!-- navigateToArrayPage -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="navigateToBasicOperationInListTryEditor" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToBasicOperationInListTryEditor -->
  <system-out/>
  <testcase classname="dsAlgoTests.ArrayTest" name="navigateToArrayInPython" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToArrayInPython -->
  <system-out/>
</testsuite> <!-- dsAlgoTests.ArrayTest -->
